job-server-python/src/python/sparkjobserver/subprocess.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import print_function
+
+import sys
+from importlib import import_module
+from py4j.java_gateway import JavaGateway, java_import, GatewayClient
+from pyhocon import ConfigFactory
+from pyspark.context import SparkContext, SparkConf
+from pyspark.sql import SQLContext, HiveContext
+from sparkjobserver.api import ValidationProblem
+
+def exitWithFailure(message, exitCode = 1):
```

Comment:
Shall we have consistency and not use camel cases for method names and variables.